src/main/perl/Log.pm

```diff
@@ -18,6 +18,8 @@ use Readonly;
 Readonly our $FH => 'FH';
 Readonly our $FILENAME => 'FILENAME';
 Readonly my $TSTAMP => 'TSTAMP';
+# $PID exists 'use English;' (it's $$)
```

Comment:
i clarified the comment to 
`# variable $PID exists when using 'use English;' (and it is $$)`
but it's for people wonderng why i'm not using `$PID` for this readonly.